contents/handbook/growth/marketing/open-source-sponsorship.mdx

```diff
@@ -57,6 +57,11 @@ In addition to sponsoring key projects, we also provide a $100/month budget for
 | [Webdriver manager for python]                       | [SergeyPirogov]   | Powers parts of our subscriptions and exports                                          | [GitHub Sponsors][github-sponsors]      | $15                     |
 | [alex]                       | [wooorm]   | We use it for improving the content we write                                           | [GitHub Sponsors][github-sponsors]      | $5                     |
 | [Caddy]                       | [mholt]   | We recommend it as a [reverse proxy](/docs/advanced/proxy#deploying-a-reverse-proxy).                                      | [GitHub Sponsors][github-sponsors]      | $50                     |
+| [Tiptap]                       | [ueberdosis]   | The headless editor framework for web artisans.                             | [GitHub Sponsors][github-sponsors]      | $149                     |
+
+
+
+https://github.com/sponsors/ueberdosis/sponsorships?sponsor=PostHog&tier_id=92321&preview=false
```

Comment:
Don't think this link needs to be in here

Reply:
```suggestion
```